src/components/button/Button.TypeTest.tsx

```diff
@@ -0,0 +1,81 @@
+// eslint-disable @typescript-eslint/ban-ts-comment
+
+import React from 'react';
+
+import { ShareIcon } from '../icons';
+
+import Button from './index';
+
+const _Button = () => {
+  return (
+    <div>
+      {/* @ts-expect-error children is missing */}
+      <Button onClick={() => null} />
+      {/* @ts-expect-error href not allowed */}
+      <Button href="foo/bar" onClick={() => null}>
+        Label
+      </Button>
+      {/* @ts-expect-error isExternal not allowed */}
+      <Button isExternal={true} onClick={() => null}>
+        Label
+      </Button>
+      <Button>Label</Button>
+      <Button onClick={() => null}>Label</Button>
+    </div>
```

Comment:
Maybe you can add this case -> `<Button href="asdasdsa" isExternal={true}>` where we expect an error

Reply:
we are looking at the normal button variant here, not the link. `href` and `isExternal` are both covered on line 19 and 15